examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_edge_result.status)
+        elif trigger_type == 2:
+            trigger_hysteresis_result = scope_service.ConfigureTriggerHysteresis(niscope_types.ConfigureTriggerHysteresisRequest(
+            vi = vi,
+            trigger_source = "0",
+            level = 0.0,
+            hysteresis = 0.1,
+            slope = niscope_types.TriggerSlope.TRIGGER_SLOPE_NISCOPE_VAL_POSITIVE,
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_hysteresis_result.status)	
+        elif trigger_type == 3:
+            trigger_digital_result = scope_service.ConfigureTriggerDigital(niscope_types.ConfigureTriggerDigitalRequest(
+            vi = vi,
+            trigger_source = "VAL_PFI_1",
+            slope = niscope_types.TriggerSlope.TRIGGER_SLOPE_NISCOPE_VAL_POSITIVE,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_digital_result.status)	
+        elif trigger_type ==  4:
+            trigger_window_result = scope_service.ConfigureTriggerWindow(niscope_types.ConfigureTriggerWindowRequest(
+            vi = vi,
+            trigger_source = "0",
+            low_level = -0.1,
+            high_level = 0.1,
+            window_mode = niscope_types.TriggerWindowMode.TRIGGER_WINDOW_MODE_NISCOPE_VAL_ENTERING_WINDOW,	
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_window_result.status)
+        elif trigger_type == 5:
+            trigger_immediate_result = scope_service.ConfigureTriggerImmediate(niscope_types.ConfigureTriggerImmediateRequest(
+            vi = vi
+            ))
+            CheckForError(scope_service, vi, trigger_immediate_result.status)
+
+        if (previous_min_sample_rate != min_sample_rate or previous_max_input_frequency != max_input_frequency or previous_min_record_length != num_records ):
+            # Use NI-TClk to configure appropriate parameters, synchronize digitizers, and initiate operation.
+            homogenous_trigger_result = tclk_service.ConfigureForHomogeneousTriggers(nitclk_types.ConfigureForHomogeneousTriggersRequest(
+                sessions = sessions
+            ))
+            CheckForError(tclk_service, vi, homogenous_trigger_result.status)
+            
+            synchronize_result = tclk_service.Synchronize(nitclk_types.SynchronizeRequest(
+                sessions = sessions,
+                min_tclk_period = 0.0	
```

Comment:
If we hard code this min_tclk_period, then we need to either test it once with real hardware or confirm with driver experts if it will not cause any issues for other model numbers of scope. In LabVIEW example, I remember that they used to not pass any value for this. But did C example pass it as 0?